Packages

zoli opened the pull request:
This should be merged after https://github.com/limetext/lime-backend/pull/84 pr merged to backend

zoli committed:
some tweaks, preventing errors

zoli committed:
moved ed.Init() after adding OnNewWindow event

zoli committed:
working with new lime-packages

zoli committed:
using new lime-packages

erbridge commented on main/qml_frontend.go:
```diff
@@ -170,7 +170,7 @@ func (t *qmlfrontend) onNew(v *backend.View) {
 		}
 		item.Set("myView", fv)
-		item.Set("fontSize", float64(v.Settings().Get("font_size", 12).(int)))
+		item.Set("fontSize", float64(12))
```
Why did this change?

zoli replied:
This was because `v.Settings().Get("font_size", 12)` is sometimes returning int sometimes float64.
A better solution added by 6b5cc64

zoli committed:
temporary workaround for settings problem

v.Settings().Get("font_size", 12) is sometimes returning int sometimes
float64.
